Various fixes following issues in Sandbox

blairboy362 opened the pull request:
Tags, security group rules and some malformed concourse yaml.

blairboy362 committed:
Various fixes following issues in Sandbox

tlwr commented on modules/k8s-cluster/security.tf:
```diff
@@ -8,10 +8,10 @@ resource "aws_security_group" "controller" {
-    "kubernetes.io/cluster/${var.cluster_name}" = "owned"
-  }
+  tags = "${map(
+    "Name", "${var.cluster_name}-controler",
```
```suggestion
    "Name", "${var.cluster_name}-controller",
```

philandstuff commented on modules/k8s-cluster/security.tf:
```diff
@@ -109,6 +109,28 @@ resource "aws_security_group_rule" "nodes-from-vpc" {
+
+  source_security_group_id = "${aws_security_group.node.id}"
+}
+
```
I deliberately didn't replicate these rules when introducing the new security group because `nodes-from-controller` is already covered by `nodes-from-vpc` and `controller-to-nodes` is covered by `controller-egress`.

philandstuff replied:
🤔 somehow the `controller-egress` rule doesn't seem to exist in the sandbox console but i definitely saw terraform create it.

philandstuff committed:
fix typo

philandstuff committed:
Remove unneeded security group rules

These rules are analogous to similar ones defined in the original
nodegroup.yaml cloudformation template.  I deliberately didn't
replicate these rules when introducing the new security group because
`nodes-from-controller` is already covered by `nodes-from-vpc` and
`controller-to-nodes` is covered by `controller-egress`.
